ocs_ci/ocs/resources/pod.py

```diff
@@ -12,7 +12,8 @@
 import base64
 
 from ocs_ci.ocs.ocp import OCP
-from ocs_ci.ocs import constants, defaults, workload
+from ocs_ci.ocs import workload
```

Comment:
where is the usage of this?

Reply:
constants and defaults where imported twice just removed it